### DISCUSSION TITLE
Is it possible to make a connection between two ESP32 using the "initiator" method?

### DISCUSSION THREAD

**Discussion by AtalibaJunior** [lathoub/Arduino-AppleMIDI-Library#161] [Q&A]
I tried to use the "initiator" method on an esp, but I didn't succeed. 
I'm using w5500 on both microcontrollers.

**Comment by lathoub** [comment]
Can you indicate if the issue was resolved (and how) - for others to learn from (and me :-))

**Reply by AtalibaJunior** [reply]
Yes!
The aim was to make the connection between two esp32 using the w5500 modules. I used the examples from the library, which by the way are working perfectly and without bugs.
The problem was with me. hahahaha
With my lack of attention, I used the same mac value on both microcontrollers. So the RTP session connection was not successful. When I changed the mac value on one side, everything worked.
I apologize for the horrible English, as I am using a translator. hahahahaha
